test/test_foreach.py

```diff
@@ -927,32 +669,34 @@ def test_unary_op_tensors_on_different_devices(self, device, dtype, op):
         else:
             self.assertEqual(expected, tensors)
 
-    @dtypes(*torch.testing.get_all_dtypes(include_bfloat16=True))
-    def test_binary_op_tensors_on_different_devices(self, device, dtype):
+    @dtypes(*torch.testing.get_all_dtypes())
```

Comment:
you probably don't need to run this test for all the dtypes, if the goal is just to error out for the different devices

Reply:
sounds good and unary version doesn't use this decorator, thanks.